assets/css/style.css

```diff
@@ -466,6 +489,14 @@ input {
     width: 569px;
   }
 
+  .article-item button,
+  .about-me-content button {
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    border-radius: 8px;
+  }
```

Comment:
- [ ] Good job making the changes @WilfriedM01, Although I notice the changes are only reflected for the desktop design If you notice on the Figma design these changes are part of the mobile design too. Please consider applying the changes in both layouts.